src/main/java/lavalink/client/io/jda/JdaLavalink.java

```diff
@@ -139,6 +139,8 @@ public void onEvent(@NonNull GenericEvent event) {
             link.removeConnection();
         } else if (event instanceof ChannelDeleteEvent) {
             ChannelDeleteEvent e = (ChannelDeleteEvent) event;
+            if(!e.isFromType(ChannelType.VOICE)) return;
```

Comment:
shouldn't this be voice or stage? or better just check for the `AudioChannel` interface